src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+**AppConfig**
```

Comment:
please make this part above 2.0.80

Reply:
done

Reply:
please rebase to dev first and it should be at the top of history.rst

Reply:
Done.